salt/orch/removal.sls

```diff
@@ -381,7 +381,8 @@ remove-target-salt-key:
   {%- endif %}
 
   {%- set affected_expr = 'G@bootstrap_complete:true' +
-                          ' and not P@.*_in_progress:true' +
+                          ' and not G@update_in_progress:true' +
```

Comment:
Missing bootstrap_in_progress:true ?

Reply:
Yeah... same as the previous one. Fixing.